packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -182,14 +182,6 @@ const baseConfig = (target) => {
                     },
                     ...(target === 'web' ? {fallback: {crypto: false}} : {})
                 },
-                // To add a new config for an extension, you can add a new element to the plugins array with the key and value as follows:
-                // The key should be the name of the plugin, and the value should be a boolean.
-                // The value should be true if the plugin is enabled, and false if it is disabled.
-                // The key should start with SFDC_EXT_, so that it's clear that it is an extension and not part of the core PWA Kit.
-                // For example, to add a new config for the Hello World extension, you can add the following:
-                // new webpack.DefinePlugin({
-                //     SFDC_EXT_HELLO_WORLD_ENABLED: true
-                // })
```

Comment:
This is no longer needed.

Reply:
What replaces this as the source of truth for defining an enum / dictionary of all active or available plugins? We should define this in a README